src/main/java/org/jenkinsci/test/acceptance/po/Control.java

```diff
@@ -155,14 +155,7 @@ public String get() {
      * Any existing value gets cleared.
      */
     public void set(@Nullable String text) {
-        // if the text is longer than 255 characters, use the high throughput variant
-        if (text != null && text.length() > 255) {
-            setAtOnce(text);
-        } else {
-            WebElement e = resolve();
-            e.clear();
-            e.sendKeys(StringUtils.defaultString(text));
-        }
+        setAtOnce(text);
```

Comment:
Somehow `sendKeys()` is busted and sends something other than the text, apparently a bogus path in `/tmp`. This wasn't exposed before this PR because the path was above 255 characters, triggering the "high throughput variant" that avoids the bug.